code/modules/mob/living/simple_animal/hostile/terror_spiders/white.dm

```diff
@@ -20,6 +20,7 @@
 	melee_damage_lower = 5
 	melee_damage_upper = 15
 	spider_tier = TS_TIER_2
+	move_resist = MOVE_FORCE_STRONG
```

Comment:
Spoke to Kyet on discord. Please just scope this on the top level terror class and remove the overrides on the subtypes.

More consistency and less shitcode are good things.

Reply:
Fair enough 👍

Reply:
It was done on purpose in #15062 just as a note